org.hl7.fhir.validation.cli/src/main/java/org/hl7/fhir/validation/cli/tasks/RePackageTask.java

```diff
@@ -93,4 +119,49 @@ private static ExpansionPackageGeneratorOutputType getExpansionPackageGeneratorO
     }
     return ExpansionPackageGeneratorOutputType.FOLDER;
   }
+
+  private List<CanonicalResource> loadResources(List<String> canonicals, FilesystemPackageCacheManager pcm) {
```

Comment:
Please check the `search` method of the PackageClient utility class. I think this method is manually constructing a query that's already handled there.